Ports/OpenJDK/package.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env -S USE_CCACHE=false bash ../.port_include.sh
+
+port="OpenJDK"
+version="17.0.2"
+workdir=jdk17u-dev-jdk-17.0.2-ga
+useconfigure=true
+use_fresh_config_guess=true
+config_guess_paths=("make/autoconf/build-aux/autoconf-config.guess")
+use_fresh_config_sub=true
+config_sub_paths=("make/autoconf/build-aux/autoconf-config.sub")
+auth_type=sha256
+files="https://github.com/openjdk/jdk17u-dev/archive/refs/tags/jdk-17.0.2-ga.tar.gz jdk-17.0.2-ga.tar.gz cb5b2a5d0916723d340f2c5bacd4607f8b8dc3a18dc8019fcfabf5306e2a4112"
```

Comment:
Please try to use `${version}` as much as possible

Reply:
good point. for major versions we'll still have to edit the path here.